tests/conftest.py

```diff
@@ -203,6 +203,10 @@ def dummy_litter_data(layer_roles_fixture):
         [0.6, 0.31, 0.02], dims=["cell_id"]
     )
     """Below ground structural litter pool (kg C m^-2)"""
+    data["excess_excrement"] = DataArray(
+        [8e-07, 8.42857e-07, 3.28571e-05], dims=["cell_id"]
+    )
+    """Rate of excrement input from the animal model [kg C m^-2 day^-1]."""
```

Comment:
Out of interest, where did these numbers come from?

Reply:
These numbers are basically just invented, I could try and estimate something from data if you think that would be better

Reply:
Nah, that's fine. I might add a comment saying these numbers are totally made up.